modules/Calendar4You/CalendarUtils.php

```diff
@@ -441,12 +441,14 @@ function transferForAddIntoTitle($type, $row, $CD) {
 		$value = getTranslatedString($Cal_Data[1],'Calendar');
 	else
 		$value = $Cal_Data[1];
-
+	if ($CD["fieldname"] == "location")  {
+		$value = "<a href='https://www.google.nl/maps/place/".$row['location']."' target='_blank'>".$row['location']."</a>";
+	}
 	if ($type == "1")
 		return $Cal_Data[1];
 	else
-//		return "<br><b>".$Cal_Data[0]."</b>: ".$value;
-		return '<table><tr><th>'.$Cal_Data[0].':</th><td onmouseover="vtlib_listview.trigger(\'cell.onmouseover\', $(this))" onmouseout="vtlib_listview.trigger(\'cell.onmouseout\', $(this))">'.$value.'</td></tr></table>';
+		return '<br><b>'.$Cal_Data[0].'</b>: <span onmouseover="vtlib_listview.trigger(\'cell.onmouseover\', $(this))" onmouseout="vtlib_listview.trigger(\'cell.onmouseout\', $(this))">'.$value.'</span>';
```

Comment:
This breaks the tooltip feature. We changed from the direct text to the table when we added support for tooltips in the calendar event. You can define a tooltip for accounts (for example) and when you hover your mouse over the account name in the event it will show you the details of the account. Tooltip needs the table to work correctly, so to be able to apply this change I would need you to make an individual PR with some changes to tooltip to work correctly.